Sources_v3/StreamChatUI/Common Views/AvatarView.swift

```diff
@@ -14,6 +13,10 @@ open class AvatarView: UIView {
         return imageView
     }()
     
+    override open var intrinsicContentSize: CGSize {
+        .init(width: 40, height: 40)
+    }
+    
```

Comment:
This should be configurable using the default appearance mechanism. Where exactly does this size come from?

Reply:
1. That's the size of channel avatar taken from the design of a channel list 
2. The `intrinsicContentSize` has only **{ get }** accessor so we cannot set it like we do for colors, etc. 

Possible workaround is:
```swift
open var defaultIntrinsicContentSize = CGSize(wight: UIView.noIntrincisMetrics, height: UIView.noIntrincisMetrics)

override open var intrinsicContentSize: CGSize {
  defaultIntrinsicContentSize
}

public static func initialAppearanceSetup(_ view: AvatarView) {
  view.defaultIntrinsicContentSize = .init(wight: 40, height: 40)
}
```

Reply:
@evsaev Yes, that's exactly what I meant 👍

Reply:
Ideally, `defaultIntrinsicContentSize` should be `CGSize?` and when it's not set `super.intrinsicContentSize` is used.

Reply:
fixed